recipes/rangeless/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.20201117":
```

Comment:
Could you please create an issue asking the upstream project to release versions from time to time?

Reply:
I would rather not for 2 reasons:
1. CCI allows commit snapshot based versioning perfectly fine, no need to complicate this for now.
2. It seems like the author hasn't been maintaining that project. It won't add too much value (you are probably **only** going to get a 1.0.0) to properly version it.